es/i18n/es.json

```diff
@@ -146,6 +146,41 @@
         "no-travel-hint": "Para ver más, cambia los filtros de arriba o registra algunos viajes!"
     },
 
+    "multilabel":{
+        "walk": "Caminando",
+        "e-bike": "e-bicicleta",
+        "bike": "Bicicleta",
+        "bikeshare": "Bicicleta compartida",
+        "scootershare": "Motoneta compartida",
+        "drove_alone": "Coche de Gas, Condujo solo",
+        "shared_ride": "Coche de Gas, Condujo con otros",
+        "e_car_drove_alone": "e-coche, Condujo solo",
+        "e_car_shared_ride": "e-coche, Condujo con otros",
+        "moped": "Ciclomotor",
```

Comment:
`moped` is not a standard label and should be removed.

Reply:
Removed in https://github.com/e-mission/e-mission-translate/pull/47/commits/dc888e077477e567ae6807795eda43d5edd6badf